src/app/components/auth/auth.component.scss

```diff
@@ -1,115 +1,206 @@
-@import './variables.scss';
-@import './mixins.scss';
+@import "styles/variables";
+
+.auth-light-link {
+  &:hover {
+    color: $highlight;
+  }
+}
+
+.fg-pass {
+  font-size: 0.8em;
+}
 
 .auth-container {
+  transition: all 0.2s ease-in-out;
+}
+
+.auth-home {
+  padding-top: 0px;
+  margin-top: 5px;
+}
+
+img.auth-logo {
+  width:90px;
+}
+
+.graphic-cont,
+.auth-cont {
+  display: inline-block;
```

Comment:
Do it like this, `padding: 10vh 40px 40px;`

Reply:
Done